### PR TITLE
Better conversion for the subclassing model and code reformat.

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: tutorial/TensorFlow_Keras_EfficientNet.ipynb [hunk 1]
```diff
@@ -70,36 +70,21 @@
...
    "source": [
+    "import numpy as np\n",
     "import efficientnet.tfkeras as efn\n",
-    "from keras.applications.imagenet_utils import decode_predictions\n",
-    "from efficientnet.keras import center_crop_and_resize, preprocess_input\n",
+    "from tensorflow.keras.applications.imagenet_utils import decode_predictions, preprocess_input\n",
+    "from efficientnet.preprocessing import center_crop_and_resize\n",
     "from skimage.io import imread\n",
-    "model = efn.EfficientNetB7(weights='imagenet')"
+    "model = efn.EfficientNetB0(weights='imagenet')"
```
Comment (jiafatom) [inline tutorial/TensorFlow_Keras_EfficientNet.ipynb:87]: Not using EfficientNetB7?

Comment (wenbingl) [reply tutorial/TensorFlow_Keras_EfficientNet.ipynb:87]: too slow for user trying B7 end to end. Given it's a tutorial, it should be simple and fast.

File: keras2onnx/topology.py [hunk 2]
```diff
@@ -299,37 +299,38 @@ def convert_topology(topology, model_name, doc_string, target_opset, channel_fir
...
-                '{} nchw_inputs does not make effect. Please set nchw_inputs to empty.'.format(onnx_not_imported))
-        k2o_logger().warning('{} so the convertor optimizer is not enabled.'.format(onnx_not_imported))
-    except Exception as e:
-        # either optimizer issue or converter issue, we just let it go to diagnose the issue from the converted model.
-        k2o_logger().warning('There is an error({}) happened during optimizing on the converted model!'.format(type(e)))
-        k2o_logger().warning(str(e))
-        import traceback
-        tb = traceback.format_exc()
-        k2o_logger().warning(tb)
+    if not topology.debug_mode:
```
Comment (jiafatom) [inline keras2onnx/topology.py:302]: topology.debug_mode will skip optimization?

Comment (wenbingl) [reply keras2onnx/topology.py:302]: Yes, the optimization makes the final model to be hard for debugging.

**Pull request merged**